contracts/Claimable.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity ^0.4.0;
+import './Ownable.sol';
+
+/*
+ * Claimable
+ * Extension for the Ownable contract, where the ownership needs to be claimed
+ */
+
+contract Claimable is Ownable {
+  address public pendingOwner;
+
+  modifier onlyPendingOwner() {
+    if (msg.sender == pendingOwner)
+      _;
+  }
+
+  function transfer(address newOwner) onlyOwner {
+    pendingOwner = newOwner;
+  }
```

Comment:
why not directly set owner here, like in Ownable?
I don't see the advantage of having a 2-step change of ownership.